ios/Framework/Core/Context/Storage/BaseCredentialsStoreKeyChain.swift

```diff
@@ -78,14 +80,41 @@ import UIKit
 		return true
 	}
 
-	open func loadStoredCredentials() -> Bool {
+	open dynamic func loadStoredCredentials() -> Bool {
 		return loadStoredCredentials(shouldLoadServer: false)
 	}
 
-	open func loadStoredCredentialsAndServer() -> Bool {
+	open dynamic func loadStoredCredentialsAndServer() -> Bool {
 		return loadStoredCredentials(shouldLoadServer: true)
```

Comment:
Do we need to be able to override these methods from xamarin?

Reply:
You can call super and then store the credentials somewhere else in Xamarin, I don't know 🤷‍♀️  why @victorg1991 ?

Reply:
I didn't see this message, I meant that this class won't be override in xamarin, right? You only need to override this class if you want to create a new CredentialsStore